docs/installation/local.md

```diff
@@ -80,13 +80,8 @@ For ease of development, you should create Postgres user with the same username
 ```sql
 CREATE USER john WITH PASSWORD 'start';
 CREATE DATABASE oevent WITH OWNER john;
```

Comment:
Let's remove john and start from everywhere in the project 

Use open_event_user for user, opev_pass for password and open_event for database name